<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lightEventUi</class>
 <widget class="QDialog" name="lightEventUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Temps : </string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QSpinBox" name="spMinutes">
     <property name="maximum">
      <number>60</number>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Minutes</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QSpinBox" name="spSecondes">
     <property name="maximum">
      <number>60</number>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Secondes</string>
     </property>
    </widget>
   </item>
   <item row="6" column="5">
    <widget class="QSpinBox" name="spDixiemme">
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="6" column="6">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>1/10e</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="7">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QRadioButton" name="rdState00">
     <property name="text">
      <string>0 - Melinda</string>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QRadioButton" name="rdState01">
     <property name="text">
      <string>1 - Fanny</string>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QRadioButton" name="rdState02">
     <property name="text">
      <string>2 - Juliette</string>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QRadioButton" name="rdState03">
     <property name="text">
      <string>3 - Clémence</string>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QRadioButton" name="rdState04">
     <property name="text">
      <string>4 - Noémie</string>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QRadioButton" name="rdState05">
     <property name="text">
      <string>5 - Alice</string>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="3" colspan="2">
    <widget class="QRadioButton" name="rdState06">
     <property name="text">
      <string>6 - Mathilde</string>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="3" colspan="2">
    <widget class="QRadioButton" name="rdState07">
     <property name="text">
      <string>7 - Maylis</string>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="3" colspan="2">
    <widget class="QRadioButton" name="rdState08">
     <property name="text">
      <string>8 - Edith</string>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="3" colspan="2">
    <widget class="QRadioButton" name="rdState09">
     <property name="text">
      <string>9 - Lara</string>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="3" colspan="2">
    <widget class="QRadioButton" name="rdState10">
     <property name="text">
      <string>10 - Eleia</string>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="3" colspan="2">
    <widget class="QRadioButton" name="rdState11">
     <property name="text">
      <string>11 - Livia</string>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QRadioButton" name="rdState12">
     <property name="text">
      <string>12 - Clea</string>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>lightEventUi</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>lightEventUi</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
